relay-server/src/utils/multipart.rs

```diff
@@ -297,11 +297,23 @@ pub struct ConstrainedMultipart(pub Multipart<'static>);
 impl FromRequest<ServiceState> for ConstrainedMultipart {
     type Rejection = Remote<multer::Error>;
 
-    async fn from_request(
-        request: Request,
-        _state: &ServiceState,
-    ) -> Result<Self, Self::Rejection> {
-        multipart_from_request(request).map(Self).map_err(Remote)
```

Comment:
How about passing `constraints` to this function, the other caller can pass default constraints, this one can set the size limit?